specs/commandline/policy.md

```diff
@@ -0,0 +1,696 @@
+| `level`             | Optional  | `--verification-level`               | `"strict"`    |
+| `override`          | Optional  | `--custom-level`                     | nil           |
+| `trustedIdentities` | Optional  | `--x509-id` or `--x509-cert` or both | `["*"]`       |
+
+```shell  
+notation policy add --name wabbit-network-dev --scope "dev.wabbitnetworks.io/net-monitor" --trust-store "ca:wabbit-network-dev" --x509-id "C=US, ST=WA, L=Seattle, O=Example, OU=Dev, CN=wabbit-networks.io" --verification-level "strict"
+```
+
+The execution of `add` fails in any of below cases:
+
+- The trust policy name exists.
+- More than one trust policy that uses a global scope, that is, the value of `registryScopes` is `["*"]`.
+- The values of `--x509-id` or `--x509cert` overlap. For example, the following two identity values are overlapping:
+  - "C=US, ST=WA, O=wabbit-network.io, OU=org1"
+  - "C=US, ST=WA, O=wabbit-network.io"
+
+Upon successful execution, the added trust policy is printed out. For example:
+
+In text format
+
+```text
+name: "wabbit-networks-dev"
+registryScopes: ["dev.wabbitnetworks.io/net-monitor"]
+level: "strict"
+trustStores: ["ca:wabbit-networks-dev"]
+trustedIdentities: ["x509.subject: C=US, ST=WA, L=Seattle, O=Example, OU=Dev, CN=wabbit-networks.io"]
+```
+
+In json format
+
+```json
+{
+  "name": "wabbit-networks-dev",                              
+  "registryScopes": [ "dev.wabbitnetworks.io/net-monitor" ],                                  
+  "signatureVerification": {                                  
+      "level": "strict"
+  },
+  "trustStores": [ "ca:wabbit-networks-dev" ],                
+  "trustedIdentities": [                                      
+      "x509.subject: C=US, ST=WA, L=Seattle, O=Example, OU=Dev, CN=wabbit-networks.io"
+  ]
+}
+```
+
+Users can also use `notation policy list` to confirm the trust policies are added.
+
+### Add a trust policy with default values
+
+Based on the above table, users can leveraging default values when adding a trust policy. Note that the default value of  `registrySopes` is `["*"]` called global scope, which means this policy applies to all the artifacts. There can only be one trust policy that uses a global scope.
+
+```shell
+notation policy add --name wabbit-network-dev --trust-store "ca:wabbit-network-dev"
+```
+
+Upon successful execution, the added trust policy is printed out. For example:
+
+In text format
+
+```text
+name: "wabbit-networks-dev"
+registryScopes: ["*"]
+level: "strict"
+trustStores: ["ca:wabbit-networks-dev"]
+trustedIdentities: ["*"]
+```
+
+In json format
+
+```json
+{
+  "name": "wabbit-networks-dev",                              
+  "registryScopes": [ "*" ],                                  
+  "signatureVerification": {                                  
+      "level": "strict"
+  },
+  "trustStores": [ "ca:wabbit-networks-dev" ],                
+  "trustedIdentities": [                                      
+      "*"
+  ]
+}
```

Comment:
IMO we don't need to display both json and text format for each variation of a command, its making doc large and doesnt adds much value.